src/ui/util/processLiveGameEvents.baseball.tsx

```diff
@@ -383,7 +386,7 @@ export const getText = (
 			} else if (event.throw) {
 				text = `${getName(
 					event.pid,
-				)} beats the throw and is safe at ${getBaseName(event.to)}`;
+				)} beats the throw and is safe at ${getBaseName(event.to)} (${playersByPid[event.pid].seasonStats["sb"]})`;
```

Comment:
I think this will fail because "sb" needs to be added to `seasonStatsKeys` in loatTeams.ts